src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -51,993 +34,148 @@ def tearDown(self):
-
-    request = {'registrationRequest': config['registrationRequests']}
-    responses = self._sas.Registration(request)['registrationResponse']
-    # Check registration responses.
-    cbsd_ids = []
-    self.assertEqual(len(responses), len(config['registrationRequests']))
-    for i in range(len(responses)):
-      response = responses[i]
-      expected_response_codes = config['expectedResponseCodes']
-      logging.debug('Expecting to see response code in set %s in response: %s',
-                      expected_response_codes, response)
-      self.assertIn(response['response']['responseCode'],
-                      expected_response_codes)
-      if response['response']['responseCode'] == 0:  # SUCCESS
-        self.assertTrue('cbsdId' in response)
-        cbsd_ids.append(response['cbsdId'])
-      else:
-        self.assertFalse('cbsdId' in response)
-
-    del request, responses
-    config['grantRequests'][0]['cbsdId'] = cbsd_ids[0]
-    request = {'grantRequest': config['grantRequests']}
-    responses = self._sas.Grant(request)['grantResponse']
-    # Check Grant responses.
-    for i in range(len(responses)):
-      response = responses[i]
-      expected_response_codes = config['expectedGrantResponseCodes']
-      logging.debug('Expecting to see response code in set %s in response: %s',
-                      expected_response_codes, response)
-      self.assertIn(response['response']['responseCode'],
-                      expected_response_codes)
-      if response['response']['responseCode'] == 0:  # SUCCESS
-          self.assertTrue('cbsdId' in response)
-          self.assertTrue('grantId' in response)
-          grant_expire_time = datetime.strptime(response['grantExpireTime'],
-                                                '%Y-%m-%dT%H:%M:%SZ')
-      else:
-          self.assertFalse('cbsdId' in response)
-          self.assertFalse('grantId' in response)
-
-    # Check grant response
-    self.assertEqual(responses[0]['cbsdId'], cbsd_ids[0])
-    grant_id = responses[0]['grantId']
-    grant_expire_time = datetime.strptime(responses[0]['grantExpireTime'],
-                                          '%Y-%m-%dT%H:%M:%SZ')
-    del request, responses
-    # Load exclusion zone database which includes a zone that contains the CBSD location or is within 50 meters of the CBSD location
-    self._sas_admin.InjectDatabase_url(config['fss_db'])
-
-    # Trigger daily activities
-    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
-
-    # Response code 500(TERMINATED_GRANT) should be sent for Heartbeat Request
-    request = {
-        'heartbeatRequest': [{
-            'cbsdId': cbsd_ids[0], 'grantId': grant_id, 'operationState': 'GRANTED'
-        }]
-    }
-    response = self._sas.Heartbeat(request)['heartbeatResponse'][0]
-    # Check the heartbeat response
+    #Check the heartbeat response code is 500(TERMINATED_GRANT) or 501 (SUSPENDED_GRANT) 
     self.assertEqual(response['cbsdId'], cbsd_ids[0])
```

Comment:
This is not requested by the test spec.

Reply:
Removed check of   cbsdid in hearbeat response